src/altrep-sparse-string.c

```diff
@@ -45,11 +45,16 @@ SEXP alrep_sparse_string_Materialize(SEXP x) {
   }
 
   R_set_altrep_data2(x, out);
+  // R_set_altrep_data1(x, R_NilValue);
```

Comment:
I'm going to make this task a separate issues as it isn't crucial but nice to have and should be done